helm/fcp-fd-probe/templates/azure-identity.yaml

```diff
@@ -0,0 +1,3 @@
+{{- include "ffc-helm-library.azure-identity" (list . "fcp-fd-auth.azure-identity") -}}
```

Comment:
Should reference the repo name so `fcp-fd-probe.azure-identity` instead of `fcp-fd-auth.azure-identity`.